_faq.Rmd

```diff
@@ -405,95 +430,126 @@ provides a one-inventory-per-language sample. For example, UPSID represents a
+phoible_geo <- left_join(phoible_index, glottolog,
+                         by=c("Glottocode"="glottocode"))
 
-phoible.index <- left_join(phoible.index, glottolog, by=c("Glottocode"="glottocode"))
+# show the merged data
+phoible_geo %>% head() %>% kable()
 ```
 
-The merged data.
+We can then easily see how many languages there are in PHOIBLE for each
+macroarea:
 
-```{r}
-phoible.index %>% head() %>% kable()
+```{r iso_by_macroarea}
+phoible_geo %>%
+    select(ISO6393, macroarea) %>%
+    distinct() %>%
+    group_by(macroarea) %>%
+    tally(name="Number of unique isocodes") %>%
+    kable()
 ```
 
-How many data points are there (per unique isocodes) per macroarea?
-
-```{r}
-phoible.index %>%
-  select(ISO6393, macroarea, latitude, longitude) %>%
-  group_by(ISO6393) -> isocodes.by.area
+Or, we can count the total number of inventories:
 
-table(isocodes.by.area$macroarea) %>% kable()
+```
+phoible_geo %>%
+    group_by(macroarea) %>%
+    tally(name="Number of inventories") %>%
+    kable()
 ```
 
-What about the global coverage of PHOIBLE?
+Note that there are some langoids/glottocodes for which geographic information
+is unavailable (their macroarea is `NA`).
 
-```{r}
-ggplot(data=isocodes.by.area, aes(x=longitude,y=latitude)) + borders("world", colour="gray50", fill="gray50") + geom_point()
+Finally, let's look at the global distribution of languages represented in
+PHOIBLE:
+
+```{r world_map}
+ggplot(data=phoible_geo, aes(x=longitude, y=latitude)) +
+    borders("world", colour="gray50", fill="gray50") +
+    geom_point(alpha=0.5, size=1, colour="orange")
 ```
 
-Of course this simple map does not show all of the data points for languages that are not in PHOIBLE!
+Of course this does not show all of the data points for languages that are
+*not* in PHOIBLE!
 
 
 # What phonological feature set does PHOIBLE use?
 
-In addition to phoneme inventories, PHOIBLE includes distinctive feature data for every phoneme in every language. The feature system used was created by the PHOIBLE developers to be descriptively adequate cross-linguistically. In other words, if two phonemes differ in their graphemic representation, then they necessarily differ in their featural representation as well (regardless of whether those two phonemes coexist in any known doculect). The feature system is loosely based on the feature system in [@Hayes2009] with some additions drawn from [@MoisikEsling2011].
+In addition to phoneme inventories, PHOIBLE includes distinctive feature data
+for every phoneme in every inventory. The feature system was created by the
+PHOIBLE developers to be descriptively adequate cross-linguistically. In other
+words, if two phonemes differ in their graphemic representation, then they
+should necessarily differ in their featural representation as well (regardless
+of whether those two phonemes coexist in any known doculect). The feature
+system is loosely based on the feature system in @Hayes2009 with some additions
```

Comment:
note how if you omit the square brackets, the name is rendered in text and only the date is in parentheses!

Reply:
Good tip. Thanks!